src/ui/component/dropDown/dropDown.tsx

```diff
@@ -0,0 +1,24 @@
+import type { FC } from 'react'
+import { Popover } from '@/ui/component/popover/popover'
+import './dropDown.scss'
+
+export type DropDownProps = {
+  field: JSX.Element
+  options: JSX.Element
```

Comment:
The name of the property is not clear to me. I'd use the names defined by the `Popover` component you use, which I think is more meaningful, e.g. `content`.

Reply:
Renamed it using more appropriate names. Thanks for the feedback.